### PR TITLE
Chris Flack - Project Pitch(es)

### PR DESCRIPTION
**Pull request by ccflack**
Two options. I was pretty settled on the voter app, but now that I know a bit more of what my end entails, I have concerns about finding data sources I can connect with that won't require a lot of direct copy/paste or manual data entry. I think it's been bumped to a backup option, pending some feedback.

### CODE REVIEW

File: pitches/Chris-Flack-Drinking-Buddy.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+Specific APIs, integrations and technologies you expect to use.
+
+* Untapped API can be imported for initial user profile.
+
+* Other/TBD ratings sites for keyword association and comparisons for prediction. (RateBeer, BeerAdvocate, UnTapped again... There's a lot of options; I'll need to do some more research to find which would be most accessible/useful for this task.)
+
+## Why you should help me build this
+
+On a practical level: this is an application I know a lot of people would find useful on a fairly regular basis. After some initial research, there appears to be a pretty wide selection of APIs.
+
```
Comment (emtrimble) [inline pitches/Chris-Flack-Drinking-Buddy.md]: Great idea. If you haven't already you might want to check out with Jeff and Keith Smith are up to.

File: pitches/Chris-Flack-Drinking-Buddy.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+You've probably got that one friend or bartender who always knows a the right beer to recommend. With this app, you can always have access to personalized recommendations based on your tastes and history, not an aggregated score from a thousand strangers.
+
+## Audience
+
+Who will use this application?
+
+* Anyone with an interest in trying new beers, without all the trial and error of the dartboard approach.
+
+* Anyone who has poured out a $7, five-star beer they just couldn't stand.
```
Comment (emtrimble) [inline pitches/Chris-Flack-Drinking-Buddy.md]: Preach!

File: pitches/Chris-Flack-Local-Voice.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+* Provide a forum for user discussion, including the option for verified accounts from local candidates or campaign representatives ("How would x candidate respond to x issue?")
+
+## Technology
+
+Government voting records, publicly accessible civics education resources, discussion forum with verified accounts for candidates or campaign representatives to answer questions and provide feedback.
+
+## Why you should help me build this
+
+Low voter turnout and response is often an issue in local races and off-year election cycles, and a lack of accessible information is often cited as one of the leading reasons for this. A lot of this information _is_ available publicly, but it's not easily or intuitively accessible to the average voter. Parsing and presenting this available data in an intuitive, digestible format could go a long way toward improving the quality of local voters and candidates.
```
Comment (emtrimble) [inline pitches/Chris-Flack-Local-Voice.md]: This is similar to something built during our very first cohort! I like the idea a lot, and see it as a big problem that needs to be solved. Most everyone can relate to not knowing enough about local elections, and politicians/gov't would love people to know more about what they stand for. 

The Open Indy Brigade focuses on using open data and making it more digestible for the common folk. I do think this information is out there - you just might have to knock on the state house door to get it. ;) 

https://github.com/Votenuity/Final-Project---TadPeteAndrew <-- Three guys to talk to about building something like this.

Comment (dummied) [inline pitches/Chris-Flack-Local-Voice.md]: I'm always a sucker for news-related apps ... it's part of my background.

This is the bit in their app that pulled from the General Assembly API: https://github.com/Votenuity/Final-Project---TadPeteAndrew/blob/master/lib/get_legislators.rb

**Pull request closed without merging**